tests/vmi_ignition_test.go

```diff
@@ -63,7 +63,7 @@ var _ = Describe("[Serial][rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][leve
 			defer expecter.Close()
 
 			By("Checking that the VirtualMachineInstance serial console output equals to expected one")
-			resp, err := expecter.ExpectBatch(commands, timeout)
+			resp, err := tests.ExpectBatchWithValidatedSend(expecter, commands, timeout)
```

Comment:
Same as the login. `tests.ExpectBatchWithValidatedSend` is not suitable here since the expected expression is not the output of the sent command.

Reply:
I think that [here](https://github.com/kubevirt/kubevirt/blob/eee4d2550a6b4e150b1bf0f6de11b07cb1230e0d/tests/vmi_ignition_test.go#L90) we do use it with expected command as a result of a BSend.